libraries/package-deps-hash/package.json

```diff
@@ -13,12 +13,12 @@
     "build": "heft test --clean"
   },
   "devDependencies": {
-    "@microsoft/rush-stack-compiler-3.5": "workspace:*",
     "@rushstack/node-core-library": "workspace:*",
     "@rushstack/eslint-config": "workspace:*",
     "@rushstack/heft": "workspace:*",
     "@types/heft-jest": "1.0.1",
-    "@types/node": "10.17.13"
+    "@types/node": "10.17.13",
+    "@rushstack/heft-node-rig": "workspace:*"
```

Comment:
Generally I try to keep the dependencies in alphabetical order. Eventually we should use a tool to normalize these files.